apps/prairielearn/assets/scripts/instructorAiGenerateDraftsClient.ts

```diff
@@ -0,0 +1,117 @@
+import { observe } from 'selector-observer';
+
+import { onDocumentReady } from '@prairielearn/browser-utils';
+
+const LOADING_PHRASES = [
+  'Hiring more TAs...',
+  'Refining prompts...',
+  'Collapsing wave functions...',
+  'Randomizing numbers...',
+  'Sharpening pencils...',
+  'Warming up neural networks...',
+  'Optimizing learning curves...',
+  'Accessing forbidden knowledge...',
+];
```

Comment:
LMK if you think this is too cute.

Reply:
🥲 removed in f65683d. We'll just rely on the spinner that appears in the button to indicate that it's processing.